02_Ansible_Review/README.md

```diff
@@ -10,21 +10,26 @@ By the end of this section, you will understand:
 
 ## First Commands
 
-Edit (or create) `/etc/ansible/hosts` and put one or more systems in it. Use `localhost` as a starting point:
-```
+Create an ansible inventory file. We will use `localhost` as a starting point:
```

Comment:
This is just because I haven't liked it in the past when workshops have asked me to modify stuff in `/etc/`